vm/vm.go

```diff
@@ -139,7 +140,8 @@ func (pool *Pool) Create(index int) (*Instance, error) {
 }
 
 // TODO: Integration or end-to-end testing is needed.
-//  https://github.com/google/syzkaller/pull/3269#discussion_r967650801
+//
```

Comment:
That's correct, I thought it was better to fix this lint issue with this commit just because it was very small and didn't add to the complexity of the commit. Totally open to undo the change though

Reply:
Nah, seems fine :+1: